build/build-config.yml

```diff
@@ -27,6 +27,14 @@ config:
       - work-dir: "estimate-service/src/main/resources/db"
         image-name: "estimate-service-db"
 
+  - name: "builds/digit-works/backend/LOI-Service"
+    build:
+      - work-dir: "backend/LOI-Service"
```

Comment:
Keep this lowercase loi. We usually don't use uppercase in service names and directory names.

Reply:
addressed the changes